client/my-sites/site-card/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+
+import page from 'page';
+
+/**
+ * Internal Dependencies
+ */
+import { navigation, siteSelection } from 'my-sites/controller';
+import { makeLayout, render as clientRender } from 'controller';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function () {
+	page( '/site-card/:domain?', siteSelection, navigation, null, makeLayout, clientRender );
```

Comment:
It's worth noting that using this without a site slug or with a wrong site slug will bring up the "All Sites" or site selector (depending on what we render). We should likely have a different approach here, where I'd guess we render nothing in that case.

Reply:
The iframe url will be served by the server, so the possibility of a slug being wrong is pretty low but it is a very good catch! Will certainly do so!